pkg/standalone/standalone.go

```diff
@@ -813,6 +814,7 @@ func createRedisStateStore(redisHost string, componentsPath string) error {
 
 	redisStore.Metadata.Name = "statestore"
 	redisStore.Spec.Type = "state.redis"
+	redisStore.Spec.Version = "v1.0"
```

Comment:
i think this should be v1 not v1.0 ... https://docs.dapr.io/operations/support/support-versioning/#components

Reply:
Yes. This should be v1.

Reply:
updated to v1